Service/RequestMiddleware.js

```diff
@@ -1,5 +1,12 @@
 const log = require('../Config/winston');
  
-module.exports = function(req, res){ 
+module.exports = function(req, res, next){ 
     log.info(`Request ${req.method} ${req.url}, IP: ${req.ip}`);
+    
+    res.on('finish', () => {        
+        log.info(`${res.statusCode} ${res.statusMessage}; ${res.get('Content-Length') || 0}b sent`);
+        log.info(`${res.statusCode} ${res.statusMessage}; ${res.get('Content-Length') || 0}b sent for Request ${req.method} ${req.url}, IP: ${req.ip} `);
```

Comment:
you can generate unique id for each request and include it in response
also it's useful to measure time it takes to process request